src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -109,72 +118,53 @@ const NavSearchBar = () => {
         iconEnd={isTruthy(searchQuery) ? clearSearchIcon : undefined}
       />
 
-      {numberOfSearchResults > 0 && (
-        <div className="nav-search-results">
-          <div className="nav-search-result-number">
-            <b>{numberOfSearchResults}</b> search results
-          </div>
+      <div
+        className={classNames("nav-dropdown nav-dropdown--search", {
+          show: searchQuery,
+        })}
+      >
+        <div className="nav-dropdown-title mb-05 mt-05">
+          <b>{numberOfSearchResults}</b> search results
+        </div>
```

Comment:
`mb-05`, `mt-05` are the class names I took from the mockup (the meaning is `margin-bottom`, `margin-top`).
I added them and few more class names to global.scss as extra utils, so that we can use them again in future as the designers do (also, it allows us not to create a new class name if we just need these styles).

```scss
// font size
.font-size--small {
  font-size: 0.9rem;
}

// opacity
.opacity-6 {
  opacity: 0.6;
}

// margins & paddings
.mt-05 {
  margin-top: 0.5rem;
}

.mb-05 {
  margin-bottom: 0.5rem;
}
```
but if you say that we should prefer move styles to element's class name, I'll do that.

Reply:
I decided to remove .opacity class names. I don't think we'll reuse them. Moreover, sometimes opacity is changes on hover, so anyway we'll need to define selector for the element.

Reply:
Sounds good.